### ISSUE TITLE
Please change pixi-layers to a new repo or make it the main branch

### ISSUE THREAD

**Issue by carrasco39** [pixijs/pixi-display#41]
Ok, so I installed this through `npm i pixi-display` and it took me a few hours to discover that actually the branch being used right now is the **Layer**  branch. So I had to run `github:pixijs/pixi-display#layers` to make things actually work. 

So please move it to a new repo or make the layer the main branch. It's non sense to keep it like this with the code examples being in a huge difference between the main branch right now.

**Comment by ivanpopelyshev** [comment]
yeah, i believe no one is uses old `pixi-display.js` anymore, its safe to move to main branch and convert it to v5.0.0-rc :)

**Comment by lambdalisue** [comment]
Please add https://github.com/pixijs/pixi-display/issues/10#issuecomment-314226251 to README as well once you've switched. It took me several hours to figure out how to use this with TypeScript.

**Comment by ivanpopelyshev** [comment]
@lambdalisue that's how i hate webpack users ;) Joking. I'll add it as soon as i move it to v5.
